components/organisms/Repositories/repositories.tsx

```diff
@@ -90,6 +90,14 @@ const Repositories = ({ repositories }: RepositoriesProps): JSX.Element => {
     setPage(1);
   }, [selectedFilter, setPage]);
 
+  const containerStyles = {
+    minWidth: "90%", // Adjust the value as needed for smaller screens
+  };
+
+  const smallScreenStyles = {
+    minWidth: "100%", // Set the width to occupy the full screen for mobile devices
+  };
+
```

Comment:
You don't need this... you can use tailwinds equivalent 

`w-11/12` and `w-full` 
check [tailwinds cheet sheet](https://flowbite.com/tools/tailwind-cheat-sheet/) for more utils

Reply:
Thank you for the review, @OgDev-01 

I will implement the stated changes.